mlflow/utils/autologging_utils.py

```diff
@@ -353,6 +431,11 @@ def autolog(*args, **kwargs):
         # function, allowing the integration name to be extracted from the function. This is used
         # during the execution of import hooks for `mlflow.autolog()`.
         wrapped_autolog.integration_name = name
+
+        if name in _flavor_to_module_name_and_json_key:
+            wrapped_autolog.__doc__ = (
+                gen_autologging_package_version_requirements_doc(name) + wrapped_autolog.__doc__
+            )
```

Comment:
This seems to have a strange interaction with the `@experimental` decorator. When `@experimental` and `@autologging_integration` are stacked on a method, the resulting docs look like:

<img width="1366" alt="Screen Shot 2021-02-25 at 7 37 26 PM" src="https://user-images.githubusercontent.com/39497902/109242677-2b032b00-77a1-11eb-9f70-b909c50e0158.png">

Observe that all of the method params / descriptions are now contained in a blue `Note` box, which isn't the desired behavior (only the version information should be shown in the `Note` box - see https://mlflow.org/docs/latest/python_api/mlflow.fastai.html#mlflow.fastai.autolog for reference).

To debug, you can build the MLflow docs locally by navigating to the `mlflow/docs` folder and running `make rsthtml`. This will generate html docs in the `mlflow/docs/build` folder that you can open in your browser.

Reply:
@WeichenXu123 This seems related to the indent level. I've experineced something similar before.

```
.. note:
   blah

   .. note:
       blah
```